clients/filesystem-hadoop3/src/main/java/org/apache/gravitino/filesystem/hadoop/GravitinoVirtualFileSystem.java

```diff
@@ -125,6 +132,10 @@ public void initialize(URI name, Configuration configuration) throws IOException
 
     initializeClient(configuration);
 
+    // Register the default local and HDFS FileSystemProvider
+    String fileSystemProviders = configuration.get(FS_FILESYSTEM_PROVIDERS);
+    fileSystemProvidersMap.putAll(FileSystemUtils.getFileSystemProviders(fileSystemProviders));
```

Comment:
gvfs will depend on the hadoop-catalog, can you please check the dependencies of gvfs to make sure the shading jar is correct?

Reply:
Some existing IT has checked the current code, moreover, I have verified this logic when developing python GVFS client.